homeassistant/components/google_domains/__init__.py

```diff
@@ -20,13 +20,20 @@
 
 CONFIG_SCHEMA = vol.Schema(
     {
-        DOMAIN: vol.Schema(
-            {
-                vol.Required(CONF_DOMAIN): cv.string,
-                vol.Required(CONF_USERNAME): cv.string,
-                vol.Required(CONF_PASSWORD): cv.string,
-                vol.Optional(CONF_TIMEOUT, default=DEFAULT_TIMEOUT): cv.positive_int,
-            }
+        DOMAIN: vol.All(
```

Comment:
As you pointed out yourself already in the latest comment: We no longer accept YAML configuration for integrations that connect to devices or services.

Instead, a configuration flow via the UI should be used.

For more information about this decision see:

- ADR-0010: <https://github.com/home-assistant/architecture/blob/master/adr/0010-integration-configuration.md#decision>

To get started on the integration configuration, see our developer documentation:

<https://developers.home-assistant.io/docs/config_entries_config_flow_handler>